opendkim/opendkim.conf.5.in

```diff
@@ -179,6 +179,11 @@ requires superuser access. A warning will be generated if
 .I UserID
 is not also set.
 
+.TP
+.I CheckSigningTable (Boolean)
+If set to yes, it walks the database on boot when it loads the config
```

Comment:
"the database" is of course SigningTable, but I think it is good to write the table name explicitly. (Also in opendkim.conf sample)

Reply:
corrected in commit https://github.com/trusteddomainproject/OpenDKIM/pull/228/commits/906a8b48453b1e1367e56c84ac3d5e008080c823